pontoon/base/migrations/0131_translation_active_populate.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-08-30 12:36
+from __future__ import unicode_literals
+
+import datetime
+
+from django.db import migrations
+
+
+def set_active_translations(apps, schema_editor):
+    Translation = apps.get_model('base', 'Translation')
+    print(datetime.datetime.now().time())
```

Comment:
I'll remove these `print` statements before mering. Only using that for measuring performance during development.